codes/classical/q-ary_digits/traceability.yml

```diff
@@ -0,0 +1,119 @@
+    - 'Suppose n, k, and p are as above. A threshold (secret) scheme using a hash function that is successful against pirates which decrpt with greater probability than paramter q, has the personal keys of the users consisting of (4k/3q)log(n/p) decryption keys. Note that this is the same as in the secret resilient schreme above. These types of schemes only need O(1) decryption operation performed by users to decrypt the information successfully. Finally, the amount of data redundancy overhead is 4k encrypted keys which is a large improvement compared to the above \cite{doi:10.1109/18.841169}.'
+
+  # Propose a field that you feel is applicable to a large and interesting class of codes
+  #threshold_vs_resilient_codes:
+   #- 'Suppose n is the number of users, k is the number of users known by the pirates, and p is the probability that the pirates cannot be traced. An open (public) resilient scheme using a hash function has the personal keys of the users consisting of O(k^{2}log n) descryption keys which is the amount of decryptions needed to reveal the information. The amount of data redundancy overhead is about O(k^{4}log n) \cite{doi:10.1.1.22.8278}.'
+   #- 'Suppose n, k, and p are as above. A secret resilient scheme using a hash function has the personal keys of the users consisting of O(k log(n/p)) decryption keys which is the amount of decryptions needed to reveal the information. The amount of data redundancy overhead is about O(k^{2} log(n/p)).'
+   #- 'Suppose n, k, and p are as above. A threshold (secret) scheme using a hash function that is successful against pirates which decrpt with greater probability than paramter q, has the personal keys of the users consisting of (4k/3q)log(n/p) decryption keys. Note that this is the same as in the secret resilient schreme above. These types of schemes only need O(1) decryption operation performed by users to decrypt the information successfully. Finally, the amount of data redundancy overhead is 4k encrypted keys which is a large improvement compared to the above \cite{doi:10.1.1.22.8278}.'
+
+realizations:
+  # List and explain the different "domains" of realizations in list items.
+  # Group closely related realizations into a single item. It's OK to have a
+  # single list item. Each item should more or less correspond to what would be
+  # said in a paragraph.
+  - 'Code used in broadcast messages, pay-per-view movies, protecting copyrighted online material \cite{doi:10.1109/18.915661}.'
+
+notes:
+  - 'Fully resilient schemes guarantee to trace the source of any piracy \cite{doi:10.1109/18.841169}.'
+  - 'Threshold schemes have a parameter of the scheme, say q, for which the scheme only traces sources of pirate decoders that decrypt with greater probability than q \cite{doi:10.1109/18.841169}.'
```

Comment:
@AlexanderBarg This "threshold" is different than that from quantum. Is this widespread notation? We can either put this in "rate" field, or make a new threshold-type field for this...

Reply:
Not so much widespread; I suggest to ignore.